embedded/devices/src/mppt.cpp

```diff
@@ -0,0 +1,249 @@
+#include "mppt.h"
+#define ID 0x0000
+#include <cstdint>
+#include <cstring>
+#include <iostream>
+
+mppt::mppt(Can canBus) { this->canBus = canBus; }
+
+int mppt::sendMode(uint8_t mode) {
+  return this->canBus.canSend(ID + 8, &mode, 1);
+}
+int mppt::sendMaxOutputVoltage(float maxOutputVoltage) {
+  return this->canBus.canSend(ID + 10, (uint8_t*)&maxOutputVoltage, 4);
+}
+int mppt::sendMaxInputCurrent(float maxInputCurrent) {
+  return this->canBus.canSend(ID + 11, (uint8_t*)maxInputCurrent, 4);
```

Comment:
You need to grab the address of `maxInputCurrent` with "&". This is causing the build error